src/gnome_abrt/views.py

```diff
@@ -457,36 +364,45 @@ def option_updated(self, conf, option):
 
 
     def __init__(self, application, sources, controller):
-        Gtk.ApplicationWindow.__init__(self,
-                            title=OopsWindow._TITLE,
-                            application=application)
+        super().__init__(application=application)
 
         if not sources:
             raise ValueError("The source list cannot be empty!")
 
-        self._builder = OopsWindow.OopsGtkBuilder()
-        self._builder.reset_window(self, OopsWindow._TITLE)
+        GObject.Binding.bind_property(
+                self.tbtn_search, "active",
+                self.search_bar, "search-mode-enabled",
+                GObject.BindingFlags.BIDIRECTIONAL)
+
+        label = Gtk.Label.new('')
+        label.show()
+        self.lb_problems.set_placeholder(label)
+        label.connect('map', self.placeholder_mapped, self)
+        label.connect('unmap', self.placeholder_unmapped, self)
+
+        builder = Gtk.Builder()
+        builder.set_translation_domain(GETTEXT_PROGNAME)
+        builder.add_from_resource('/org/freedesktop/GnomeAbrt/ui/oops-menus.ui')
+
+        self.menu_problem_item = builder.get_object('menu_problem_item')
+        self.menu_problem_item = Gtk.Menu.new_from_model(self.menu_problem_item)
+
+        self.menu_problem_item.attach_to_widget(self)
+
+        self.menu_multiple_problems = builder.get_object(
+                'menu_multiple_problems')
+        self.menu_multiple_problems = Gtk.Menu.new_from_model(
+                self.menu_multiple_problems)
+
+        self.menu_multiple_problems.attach_to_widget(self)
 
         #pylint: disable=E1120
         css_prv = Gtk.CssProvider.new()
-        # "row" selector is valid and supported in GTK>=3.20 (Fedora 24).
-        # "GtkListBoxRow" selector is no longer supported but required
-        # for GTK<3.20 (Fedora 23). It can be removed if we decide to stop
-        # supporting older systems.
-        css_prv.load_from_data("GtkListBoxRow, row {\n"
-                               "  padding          : 12px;\n"
-                               "}\n"
-                               ".app-name-label {\n"
-                               "  font-weight      : bold;\n"
-                               "}\n"
-                               ".oops-reason {\n"
-                               "  font-size        : 120%;\n"
-                               "  font-weight      : bold;\n"
-                               "}\n".encode()
-                               )
+        css_prv.load_from_resource('/org/freedesktop/GnomeAbrt/css/oops.css')
         stl_ctx = self.get_style_context()
         stl_ctx.add_provider_for_screen(stl_ctx.get_screen(), css_prv, 6000)
```

Comment:
(not related to this PR, but I guess this 6000 should be changed to the general css priorities constants.)

Reply:
The macros only go to 800, boss.

Reply:
The recommendation is to let users override the style by not going over 800, so I’m guessing this is intentional?

Reply:
Another idea is to just use Pango attributes and margins to achieve the same thing.

Reply:
That sounds like a bad practice, the docs say:
![image](https://user-images.githubusercontent.com/1755514/48610361-06466900-e985-11e8-8506-352789069224.png)

I think it should use the application priority after all. Anyway, not my field/call :)

Reply:
Okay, so Pango doesn’t play too well with PyGObject (attributes are unusable: https://bugzilla.gnome.org/show_bug.cgi?id=646788).

Otherwise, these particular things just aren’t meant to be themable.